leadership.html

```diff
@@ -420,95 +407,84 @@ <h1>رهبری</h1>
       <div class="row">
 
         <div class="left col">
-          <span class=title>E5: Engineering Manager</span>
-          <p><span class=focus>FOCUS: <br />Across Several Teams</span></p>
-          <a href="https://tinyurl.com/e360-eng-mgr" target="_blank" class="link underline">Role Expectations</a>
+          <span class=title>E5: مدیر مهندسی</span>
+          <p><span class=focus>تمرکز: <br />درون تیم</span></p>
+          <a href="https://tinyurl.com/e360-eng-mgr" target="_blank" class="link underline">انتظارات از این نقش</a>
         </div>
 
         <div class="middle" id="em-skills">
           <div class="facilitation show">
-            <span class="pill">Facilitation</span>
+            <span class="pill">تسهیل (Facilitation)</span>
             <ul>
-              <li>Facilitates discussions <span class="highlight">across teams</span>, ensuring that everyone has an
-                opportunity to share their opinion and be heard,
-                and that discussion outcomes tie to stated goals.
-                <br /><span class="highlight">Ensures relevant parties are included in discussions.</span>
-                <br /><span class="highlight">Guides discussions toward decisions, clarifies and gets buy-in.</span>
-                <br /><span class="highlight">Ensures decisions are communicated to stakeholders</span>
+              <li>بحث‌ها را <span class="highlight">درون تیم‌ها</span>، تسهیل می‌کند و اطمینان حاصل می‌کند همه فرصت به اشتراک گذاشتن نطرات خود و شنیده شدن را دارند
+                و اینکه نتایج بحث با اهداف بیان شده مرتبط است.
+                <br /><span class="highlight">اطمینان حاصل می‌کند که طرف‌های مربوطه در بحث‌ها شرکت می‌کنند.</span>
+                <br /><span class="highlight">
+                  بحث‌ها را به سمت تصمیم گیری هدایت می‌کند، شفاف سازی می‌کند و ؟؟.
+                  Guides discussions toward decisions, clarifies and gets buy-in.</span>
```

Comment:
@AbbasiMohamad I'd like to get your opinion about `buy-in` in this sentence.

Reply:
I don't know this either, but I searched, and it seems to give such a meaning: 

> agreement with, or acceptance of, a policy or suggestion.